descriptors/ModuleDescriptor-template.json

```diff
@@ -373,6 +373,14 @@
           "modulePermissions": [
             "circulation.check-in-by-barcode.post"
           ]
+        },
+        {
+          "methods": ["PUT"],
+          "pathPattern": "/inn-reach/transactions/{itemBarcode}/check-out-item/{servicePointId}",
+          "permissionsRequired": ["inn-reach.inn-reach-transaction.item.put"],
```

Comment:
I see that the same permission is used for another endpoint:
> "methods": ["PUT"],
          "pathPattern": "/inn-reach/transactions/{id}/receive-item/{servicePointId}",
          "permissionsRequired": ["inn-reach.inn-reach-transaction.item.put"],
          "modulePermissions": [
            "circulation.check-in-by-barcode.post"
          ]

Has it been done on purpose?

Reply:
Yes, according to requirements they both should use "permission to modify transactions"

Reply:
I'm not sure the requirement is understood correctly. Usually each endpoint has its own permission
Could you double check with PO?

Reply:
Checked

> The same one is fine